versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx

```diff
@@ -0,0 +1,33 @@
+# Setup RBAC
+
+:::note
+RBAC (Role-Based Access Control) is a feature that is only available to Enterprise customers. If you are interested in this feature, please contact our sales team at sales@langchain.dev
+Other plans default to using the Admin role for all users. Read more about roles under [admin concepts](../../concepts/admin/admin.mdx)
+:::
+
+LangSmith relies on RBAC to manage user permissions. This allows you to control who can access your LangSmith workspace and what they can do within the account.
+Only users with the `workspace:manage` permission can can manage access control.
+
+## Creating a Role
+
+While there are several default roles that come with LangSmith, you can create custom roles to suit your needs.
```

Comment:
list the roles

Reply:
would that be redundant given that we already have these listed under concepts?